pkg/auth/uaa/tanzu.go

```diff
@@ -14,10 +14,14 @@ import (
 const (
 	// Tanzu CLI client ID for UAA that has http://127.0.0.1/callback as the
 	// only allowed Redirect URI and does not have an associated client secret.
-	tanzuCLIClientID     = "tp_cli_app"
-	tanzuCLIClientSecret = ""
-	defaultListenAddress = "127.0.0.1:0"
-	defaultCallbackPath  = "/callback"
+	TanzuCLIClientID = "tp_cli_app"
+	// Alternate client ID for UAA associated with a longer refresh token
+	// validity. Use this for CLI use cases where it is impractical to
+	// interactively reauthenticated once the refresh token expires.
+	TanzuCLIClientIDExtended = "tp_cli_app" // TODO(vui) update once available.
```

Comment:
This is the only thing that prevents me from approving.  Unless we want to merge and update this when the info is available?

Reply:
Rebased. This does not apply anymore.